test/unit/basic.generate.test.js

```diff
@@ -204,11 +204,6 @@ describe('basic generate', () => {
     })
   })
 
-  test('nuxt re-generating with no subfolders', async () => {
-    generator.nuxt.options.generate.subFolders = false
-    await expect(generator.generate({ build: false })).resolves.toBeTruthy()
-  })
-
   test('/users/1.html', async () => {
```

Comment:
This test can be removed now, but please also explicitly add `nuxt.options.generate.subFolders = false` to one of the unit tests for generator. Otherwise we only have implicit tests and those will only cause problems in the long run

Actually the unit tests are somewhat confusing already because the default setting for subFolders is `true` in Nuxt, but due to how these unit tests have been setup they have different default settings. I would strongly suggest to use a single `createNuxt` method which is shared by all unit tests in all packages which have the default Nuxt settings as base on which we then extend. Similar like the loadFixture method for the fixtures.

The main reason for this is that in the real world these methods will never run without the full config anyway, so by recreating a small subsection of the nuxt config just for these tests you could unwillingly create a different use-case then the real world and thereby (partially) invalidate them.

Reply:
I added subFolders false in one test.

But for unit test, the test case should care up/downstream functionality, like unit test for `generator` does't depend on the `config` default options. This should be responsibility of fixtures for e2e tests.

Reply:
Thanks, understood :)